mcd/mcd-deplyoment/mcd-deployment.md

```diff
@@ -0,0 +1,301 @@
+---
+title: How to deploy MCD Protocol
+description: Learn how to deploy your own MCD
+parent: mcd
+tags:
+  - deployment
+  - mcd
+slug: how-to-deploy-mcd
+contentType: guides
+root: false
+---
+
+# How to deploy MCD Protocol
+
+**Level:** Intermediate  
+**Estimated Time:** 30 - 45 min
+
+- [How to deploy MCD Protocol](#how-to-deploy-mcd-protocol)
+  - [Overview](#overview)
+  - [Learning Objectives](#learning-objectives)
+  - [Pre-requisites](#pre-requisites)
+  - [Sections](#sections)
+    - [Installing](#installing)
+    - [Tokens deplyoment (Optional)](#tokens-deplyoment-optional)
+    - [Configuration](#configuration)
+      - [Account configuration](#account-configuration)
+      - [Chain configuration](#chain-configuration)
+    - [MCD Deployment process](#mcd-deployment-process)
+      - [Deploy on local testchain with default config file](#deploy-on-local-testchain-with-default-config-file)
+      - [Deploy on Goerli with default config file](#deploy-on-goerli-with-default-config-file)
+      - [Deploy on Mainnet with default config file](#deploy-on-mainnet-with-default-config-file)
+      - [Deploy on any network passing a custom config file](#deploy-on-any-network-passing-a-custom-config-file)
+      - [Output](#output)
+      - [Helper scripts](#helper-scripts)
+  - [Smart Contract Dependencies](#smart-contract-dependencies)
+  - [Additional Resources](#additional-resources)
+
+## Overview
+
+This guide will walk you through deploying MCD protocol to your prefered network.
+
+## Learning Objectives
+
+Here you’ll learn how to deploy your own MCD protocol.
+
+- MCD deployment configuration
+- Deployment process
+
+## Pre-requisites
+
+Knowledge in:
+
+- [dapp.tools](http://dapp.tools/)
+- [makerdao/dss-deploy-scripts](https://github.com/makerdao/dss-deploy-scripts)
+- Testnet ETH kovan/rinkeby/ropsten/goerli
+
+## Sections
+
+### Installing
+
+First if all we will need to clone `dss-deploy-script` repo:
+
+```shell
+git clone https://github.com/makerdao/dss-deploy-scripts.git
+cd dss-deploy-scripts
+```
+
+The only way to install everything necessary to deploy is Nix. Run
```

Comment:
No we need to add a dependency section - which would include dapp tools and NIX and links for installing those ?

Reply:
Fixed